<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDDBImport</class>
 <widget class="QDialog" name="CDDBImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import from CDDB</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Album:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_57">
       <property name="text">
        <string>Label:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>Genre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="album">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="genreId">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="genre">
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_68">
       <property name="text">
        <string>Recording device:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="labelId">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="labelName">
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="recordDeviceId">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recordDevice">
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_53">
         <property name="text">
          <string>Year of publication:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="releaseDate">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2008</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="date">
          <date>
           <year>2008</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_54">
         <property name="text">
          <string>Recording date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="recordDate">
         <property name="date">
          <date>
           <year>2008</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_67">
       <property name="text">
        <string>Recording source:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_63">
       <property name="text">
        <string>Tags:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="tags">
       <property name="maxLength">
        <number>2048</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="recordSourceId">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recordSource">
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>Artist:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="artist">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tracklist">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Track</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Artist - Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Length</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK (F12)</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/button_ok.png</normaloff>:/icons/resources/button_ok.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel (ESC)</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/cancel.png</normaloff>:/icons/resources/cancel.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>artist</tabstop>
  <tabstop>album</tabstop>
  <tabstop>releaseDate</tabstop>
  <tabstop>recordDate</tabstop>
  <tabstop>genreId</tabstop>
  <tabstop>genre</tabstop>
  <tabstop>labelId</tabstop>
  <tabstop>labelName</tabstop>
  <tabstop>recordSourceId</tabstop>
  <tabstop>recordSource</tabstop>
  <tabstop>recordDeviceId</tabstop>
  <tabstop>recordDevice</tabstop>
  <tabstop>tags</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
